make.ps1

```diff
@@ -11,15 +11,24 @@ Param(
   [string]
   $Version = "",
 
-  [Parameter(HelpMessage="Architecture (native, x64).")]
+  [Parameter(HelpMessage="The architecture to use for compiling, e.g. `"X64`", `"Arm64`"")]
   [string]
-  $Arch = "x86-64",
+  $Arch = [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture,
 
   [Parameter(HelpMessage="Directory to install to.")]
   [string]
   $Destdir = "build/install"
 )
 
+if ($Arch -ieq 'x64')
```

Comment:
'THost' is a cmake thing, which is not involved here; I thought it better to name this something different.